2021/Robot2021/src/main/cpp/commands/DriveModeToggle.cpp

```diff
@@ -10,6 +10,9 @@
 
 #include "RobotContainer.h"
 
+#include <spdlog/fmt/ostr.h>

```

Comment:
Since there aren't any variables with units in any of the print statements in the file, I'm pretty sure you don't need to include this